src/Infrastructure/Dto/TaskSetOverrideMapDto.php

```diff
@@ -0,0 +1,111 @@
+<?php
+
+/**
+ * Copyright © 2019-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Sdk\Infrastructure\Dto;
+
+class TaskSetOverrideMapDto
+{
+    /**
+     * @var array<string, bool>
+     */
+    protected array $stopOnErrorMap;
+
+    /**
+     * @var array<string, array<string>>
+     */
+    protected array $tagsMap;
+
+    /**
+     * @var array<string, array<string, string>>
+     */
+    protected array $sharedPlaceholdersMap;
+
+    /**
+     * @var array<string, array<string, array>>
+     */
+    protected array $overridePlaceholdersMap;
+
+    /**
+     * @param array<string, bool> $stopOnErrorMap
+     * @param array<string, array<string>> $tagsMap
+     * @param array<string, array<string, string>> $sharedPlaceholdersMap
+     * @param array<string, array<string, array>> $overridePlaceholdersMap
+     */
+    public function __construct(
+        array $stopOnErrorMap = [],
+        array $tagsMap = [],
+        array $sharedPlaceholdersMap = [],
+        array $overridePlaceholdersMap = []
+    ) {
+        $this->stopOnErrorMap = $stopOnErrorMap;
+        $this->tagsMap = $tagsMap;
+        $this->sharedPlaceholdersMap = $sharedPlaceholdersMap;
+        $this->overridePlaceholdersMap = $overridePlaceholdersMap;
+    }
+
+    /**
+     * @return array
+     */
+    public function getStopOnErrorMap(): array
+    {
+        return $this->stopOnErrorMap;
+    }
+
+    /**
+     * @param string $taskId
+     *
+     * @return bool|null
+     */
+    public function getTaskStopOnError(string $taskId): ?bool
+    {
+        return $this->stopOnErrorMap[$taskId] ?? null;
```

Comment:
default false not null

Reply:
By default it uses sub-task command option